Ical.Net/Evaluation/RecurrenceUtil.cs

```diff
@@ -15,7 +15,7 @@ public static void ClearEvaluation(IRecurrable recurrable)
         }
 
         public static HashSet<Occurrence> GetOccurrences(IRecurrable recurrable, IDateTime dt, bool includeReferenceDateInResults) => GetOccurrences(recurrable,
-            new CalDateTime(dt.AsSystemLocal.Date), new CalDateTime(dt.AsSystemLocal.Date.AddDays(1).AddSeconds(-1)), includeReferenceDateInResults);
```

Comment:
Converting to local time would only be correct, when dealing with Floating Time: If DTSTART does not include a TZID parameter and does not end with a Z, the event is defined in the local time of the calendar user.

Reply:
Even then `AsSystemLocal` might not be correct, because the end user's time zone might come from a completely different context than the system's time zone, in particular when using the lib on a web server. We might be processing calendars received via the web and in this case the systems local time would easily be different from the user's, depending on whether it is/can be derived form the request.

Anyhow, this seems to be a breaking change. I think it's a bugfix, therefore it should be fine. Just to be aware, that some users might have adapted to that behavior.

Related to #623, #588, #19, https://github.com/orgs/ical-org/projects/3?pane=issue&itemId=85730174

Reply:
Yes, this is potentially a breaking change, even when it's rather a fix. This must go into release notes for sure.